### ISSUE TITLE
Needs to be updated for latest koa-router?

### ISSUE THREAD

**Issue by Pitchlab** [koajs/mount#46]
The example didnt work immediately out of the box, as it would not load the router and then stumbled over the app.get() so I checked the koa-router examples. 

Changing the line at the start from
`var router = require('koa-router');`
to 
`var router = require('koa-router')();`

and having

```
app.use(router.routes());

router.get('/handle_facebook_callback', function* (next) {
  console.log(this.query);
  this.body = JSON.stringify(this.query, null, 2);
});

router.get('/handle_twitter_callback', function* (next) {
  console.log(this.query);
  this.body = JSON.stringify(this.query, null, 2);
});

```
made it run and work for twitter at least.

**Comment by halt-hammerzeit** [comment]
@Pitchlab Of course, you are supposed to call `router()` to get an instance because it's not a singleton.

**Comment by Pitchlab** [comment]
Yes, apparently, but that may be nice to reflect in the examples to save people like me some time figuring it out :) Overall it's great stuff!

**Comment by halt-hammerzeit** [comment]
I'm not a maintainer btw.
Just passing by.
Go make a PR.

**Comment by Pitchlab** [comment]
No I won't. Go pass by somewhere else.

**Comment by halt-hammerzeit** [comment]
That's called "irresponsibility"

**Comment by Pitchlab** [comment]
What exactly is your problem? I posted a notice of something I found out about, that is too small IMO to merit a pull request and that I wanted to discuss first because maybe I did something wrong. Then you made a comment, to which I replied in a nice manner, even though it was strictly speaking quite patronizing. Then you start telling me what to do, even after admitting you are just a passer by. Are you always this unhappy? Do you like being angry at people that are -in your opinion- 'wrong on the internet'? 

TL;DR: Your kind of trolling will be way more effective on youtube videos.

**Comment by Pitchlab** [comment]
No it is not irresponsible, just not worthy of a pull.

**Comment by halt-hammerzeit** [comment]
@Pitchlab 

> What exactly is your problem? I posted a notice of something I found out about, that is too small IMO to merit a pull request and that I wanted to discuss first because maybe I did something wrong. Then you made a comment, to which I replied in a nice manner, even though it was strictly speaking quite patronizing. Then you start telling me what to do, even after admitting you are just a passer by. Are you always this unhappy? Do you like being angry at people that are -in your opinion- 'wrong on the internet'?

Why so nervous.
Go get a yoga lesson.
